.github/workflows/build-app.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
 
   release:
     name: Release Build and Publish
-    if: github.event_name == 'push'
+    if: github.event_name == 'push' || github.event_name == 'pull_request'
```

Comment:
Publish to play on pull request?

Reply:
This is just so I can debug on the PR and don't need to keep committing to main/release - will be gone before merge.